src/client/datascience/export/fileConverter.ts

```diff
@@ -35,16 +35,14 @@ export class FileConverter implements IFileConverter {
     ) {}
 
     // Import a notebook file on disk to a .py file
-    public async importIpynb(contents: string, source: Uri): Promise<void> {
+    public async importIpynb(source: Uri): Promise<void> {
         const reporter = this.progressReporter.createProgressIndicator(localize.DataScience.importingIpynb(), true);
+        let nbDoc;
         try {
-            const target = await this.getTargetFile(ExportFormat.python, source);
-            if (!target) {
-                return;
-            }
-            const exportInterpreter = await this.exportInterpreterFinder.getExportInterpreter(undefined);
-            await this.performNbConvertExport(ExportFormat.python, contents, target, exportInterpreter, reporter.token);
-            await this.exportFileOpener.openFile(ExportFormat.python, target);
+            // Open the source as a NotebookDocument, note that this doesn't actually show an editor, and we don't need
+            // a specific close action as VS Code owns the lifetime
+            nbDoc = await workspace.openNotebookDocument(source);
```

Comment:
I actually thought that I would have to either mess around with the serializer here, or close down this document. But in the research that I did this doesn't seem to be the case. 
With open we don't actually open up an editor here that requires a show step. And vscode actually owns the lifetime of this opened document, we don't have to close it down it will do that for us.

https://github.com/microsoft/vscode/blob/e8667df1082844647ef48476ea0d3d523a020944/src/vs/vscode.d.ts#L11003
https://github.com/microsoft/vscode/issues/15723#issuecomment-288371863